config/99_aptget

```diff
@@ -0,0 +1 @@
+udev ALL=(ALL) NOPASSWD: ALL
```

Comment:
wouldn't it be nicer to just pipe this line directly into `/etc/sudoers.d/99_aptget` in the Dockerfile? That way you can use the ${USER} variable and don't have to have udev specified in two places. Plus it reduces the number of files needed

Reply:
great idea yes definitely